src/uv-common.h

```diff
@@ -60,6 +60,14 @@ extern int snprintf(char*, size_t, const char*, ...);
 #define STATIC_ASSERT(expr)                                                   \
   void uv__static_assert(int static_assert_failed[1 - 2 * !(expr)])
 
+#ifdef __GNUC__
+#define uv__load_relaxed(p) __atomic_load_n(p, __ATOMIC_RELAXED)
+#define uv__store_relaxed(p, v) __atomic_store_n(p, v, __ATOMIC_RELAXED)
+#else
+#define uv__load_relaxed(p) (*p)
+#define uv__store_relaxed(p, v) do *p = v; while (0)
```

Comment:
```suggestion
#define uv__load_relaxed(p) __atomic_load_n((p), __ATOMIC_RELAXED)
#define uv__store_relaxed(p, v) __atomic_store_n((p), (v), __ATOMIC_RELAXED)
#else
#define uv__load_relaxed(p) (*(p))
#define uv__store_relaxed(p, v) do *(p) = (v); while (0)
```

I just do this out of habit. No idea if it's necessary.

Reply:
I used to always write it like that as a matter of hygiene and in fact I initially did it that way for this PR (but changed it again because I didn't like the excess of parentheses) but if it stands out to you, that's good enough reason to do it.

Reply:
@vtjnash I noticed just now that I somehow managed to drop the fix-up commit when I merged this PR. It should do no harm but I can open a fix-up PR if you feel strongly.

Reply:
I don't feel strongly about it.